### ISSUE TITLE
how 2 ouput origin h w size？

### ISSUE THREAD

**Issue by York1996OutLook** [lucidrains/segformer-pytorch#9]

**Comment by talhaanwarch** [comment]
need to interpolate
`nn.functional.interpolate(logits, size=labels.shape[-2:], mode="bilinear", align_corners=False)`

**Comment by Philos01** [comment]
> need to interpolate `nn.functional.interpolate(logits, size=labels.shape[-2:], mode="bilinear", align_corners=False)`

i think it's not  a good ideal,but i don't know how can output origin h w size and don't break it's origin information

**Comment by jie666-6** [comment]
Did anyone solve this issue? I do not know how to modify the code to keep the out put with the same shape of input so far
